.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+If applicable, please include steps to reproduce the issue.
```

Comment:
in interest of keeping these short: I don't think this line adds anything not already stated. It seems implied to me